pages/tools/slack/records.md

```diff
@@ -44,6 +44,15 @@ they have been reviewed in the past. Various legal actions (for example, a
 Freedom of Information Act (FOIA) request) or security operations approved by
 the TTS Tech Portfolio Director have required others to view the exports.
 
+When Slack or other records are released under FOIA, GSA will disclose employee
+names unless disclosure would clearly violate the employee’s personal privacy. 
+This would include situations where the agency has a concrete basis to believe 
```

Comment:
In this sentence, does "This would include" refer to "will disclose" or does it refer to situations that violate personal privacy? I'm not sure which way it goes.

Reply:
Do you think it might mean this?
> GSA discloses names even in situations where the agency has a concrete basis to believe that disclosure will result in harassment

Reply:
I'll ask

Reply:
I hope it means this, but I'm asking. An answer may take weeks.
> GSA discloses employee names in FOIA productions, unless (at the time of the FOIA production) disclosure would clearly violate the employee’s personal privacy. **GSA will not disclose employee names** in situations where the agency has a concrete basis to believe that disclosure will result in harassment of the employee.